pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -1,25 +1,27 @@
-{ stdenv, fetchurl, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2}:
+{ stdenv, fetchgit, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2 }:
 
-let version = "0.2.1"; in
+let version = "0.3-pre"; in
 
 buildPythonPackage {
   name = "qutebrowser-${version}";
   namePrefix = "";
-  
-  src = fetchurl {
-    url = "https://github.com/The-Compiler/qutebrowser/releases/download/v${version}/qutebrowser-${version}.tar.gz";
-    sha256 = "b741a1a0336b8d36133603a3318d1c4c63c9abf50212919200cd2ae665b07111";
-    };
-    # Needs tox
-    doCheck = false;
 
-    propagatedBuildInputs = [
-      python pyyaml pyqt5 jinja2 pygments pypeg2
-    ];
-    
-    meta = {
-      homepage = https://github.com/The-Compiler/qutebrowser;
-      description = "Keyboard-focused browser with a minimal GUI";
-      license = stdenv.lib.licenses.gpl3Plus;
+  src = fetchgit {
+    url = "https://github.com/The-Compiler/qutebrowser.git";
+    rev = "f31f254d9bf3ffd4ef95089f4924e5c45d8c0f78";
+    sha256 = "0virh71q9qyh7ggk5p6h3gjv30gr2jn8am6yq8xzbslchck0rkcr";
+  };
+
+  # Needs tox
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    python pyyaml pyqt5 jinja2 pygments pypeg2
+  ];
+
+  meta = {
```

Comment:
Do you want to add yourself as a maintainer? If you don't want then add me please.